mappings/net/minecraft/item/Item.mapping

```diff
@@ -15,17 +15,34 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 		ARG 1 settings
 	METHOD method_16698 isNetworkSynced ()Z
 	METHOD method_19263 isFood ()Z
+		COMMENT Checks if this item is food and therefore is edible.
 	METHOD method_19264 getFoodComponent ()Lnet/minecraft/class_4174;
 	METHOD method_21830 getEatSound ()Lnet/minecraft/class_3414;
 	METHOD method_21831 getDrinkSound ()Lnet/minecraft/class_3414;
 	METHOD method_24357 damage (Lnet/minecraft/class_1282;)Z
 		ARG 1 source
 	METHOD method_24358 isFireproof ()Z
 	METHOD method_7836 use (Lnet/minecraft/class_1937;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;)Lnet/minecraft/class_1271;
+		COMMENT Called when an item is used by a player.
+		COMMENT This typically is bound to using the right mouse button.
+		COMMENT
+		COMMENT <p>This method is called on both the logical client and logical server, so take caution when using this method.
+		COMMENT The logical side can be checked using {@link net.minecraft.world.World#isClient() world.isClient()}.
+		COMMENT
+		COMMENT @return a typed action result that specifies if using the item was successful.
+		COMMENT The action result contains the new item stack that the player's hand will be set to.
```

Comment:
Make note that the use call goes from main hand to off hand and may perform hand animation based on the result